node/constants/index.ts

```diff
@@ -9,6 +9,6 @@ export enum AddressType {
 
 // Delivery channels
 export const DELIVERY = 'delivery'
-export const PICKUP_IN_STORE = 'pickup-in-store'
+export const PICKUP_IN_STORE = 'pickup-in-point'
```

Comment:
I would suggest to change the const name too.
```suggestion
export const PICKUP_IN_POINT = 'pickup-in-point'
```